docs/README.md

```diff
@@ -103,7 +103,7 @@ Sections that were moved:
 
 Use the relative style to link to the new file so that the versioned docs continue to work.
 
-For an example of a rich moved section set please see the very end of [the Trainer doc](https://github.com/huggingface/transformers/blob/main/docs/source/en/main_classes/trainer.mdx).
+For an example of a rich moved section set please see the very end of [the Trainer doc](https://github.com/huggingface/transformers/blob/main/docs/source/en/main_classes/trainer.md).
```

Comment:
This links to `transformers` docs rather than SetFit docs, where they do have `trainer.md` rather than `trainer.mdx`. However, perhaps I should link to https://huggingface.co/docs/transformers/main_classes/trainer instead? That seems most sensible.

It shouldn't matter much though, this is just the README for the docs. I made this change because I ran a "dead URL" detection program on the directory and it pointed out this issue.